README.md

```diff
@@ -1053,6 +1053,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Deploy Now](https://deploynow.space) — Deploy smarter. Deploy faster. Deploy Now. - Deploy up to 3 web projects from your GitHub repository for free.
   * [DigitalOcean](https://www.digitalocean.com/pricing) - Build and deploy 3 static sites for free on the App Platform Starter tier. 
   * [Drive To Web](https://drv.tw) — Host directly to the web from Google Drive & OneDrive. Static sites only. Free forever. One site per Google/Microsoft account.
+  * [DuckDocs](https://duckdocs.com) - Free markdown-powered documentation hosting, sort of like a hosted Docusaurus.
```

Comment:
It's not `Free` its ` Free for 10 Pages Per Site` :)

Reply:
Oh yes that's true,let me update